src/components/send/SendContainerWrapper.js

```diff
@@ -98,6 +107,7 @@ export function SendContainerWrapper({ match }) {
                     return;
                 }
 
+                Mixpanel.track("SEND token");
```

Comment:
Shouldn't we be using the same `Mixpanel.withTracking()` wrapper here around the actual attempt to perform the transfer, just as we do in the existing `Send` screen?

Without the wrapper, we will not report to Mixpanel cases where the send _fails_; we just won't send any tracking event at all, which represents a breaking change in our tracking reporting if we rely on the failures being reported for any of our mixpanel insights.